day_two_guide/topics/alerts_using_prometheus.adoc

```diff
@@ -0,0 +1,33 @@
+////
+Creating alerts using Prometheus
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_health_checks.adoc
+////
+
+While the topics in this section are for manually checking the health of an
```

Comment:
With an eye towards reuse, I'd remove or conditionalize "While the topics in this section are for manually checking the health of an {product-title} component, "